thorium/testsuite/test_serializer.py

```diff
@@ -73,7 +85,7 @@ def test_serialize_empty_detail_response(self):
             "type": "detail",
             "error": None,
             "status": 200,
-            "meta": {},
+            "meta": {'pagination': None},
```

Comment:
You can probably avoid this by deleting the attribute instead of setting it to None.